flink-connector-aws/flink-connector-aws-kinesis-streams/src/test/java/org/apache/flink/connector/kinesis/source/examples/SourceFromKinesis.java

```diff
@@ -34,7 +34,13 @@
 public class SourceFromKinesis {
 
     public static void main(String[] args) throws Exception {
-        final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+        Configuration conf = new Configuration();
+        conf.setString(
+                "metrics.reporter.slf4j.factory.class",
+                "org.apache.flink.metrics.slf4j.Slf4jReporterFactory");
+        conf.setString("metrics.reporter.slf4j.interval", "1s");
```

Comment:
Cool way to verify metrics, but let's remove this from the example..! Doesn't seem related to setting up the KinesisSource